### PR TITLE
pcsx2: update to 1.6.0

### PR DESCRIPTION
**Pull request by selfsigned**
[ci skip]
Update pcsx2 to the latest stable version, the patches are now unneeded.

As a side-note, in the template, the license is said to be ` license="GPL-2, GPL-3, LGPL-2.1, LGPL-3"`
Shouldn't it be [GPL-2](https://github.com/PCSX2/pcsx2/blob/master/COPYING.GPLv2)?

### CODE REVIEW

File: srcpkgs/pcsx2/template [hunk 1]
```diff
@@ -4,19 +4,20 @@ lib32mode="full"
 nopie=yes
 
 pkgname=pcsx2
-version=1.4.0
-revision=7
+version=1.6.0
+revision=8
```
Comment (abenson) [inline srcpkgs/pcsx2/template:8]: Revision should be 1

File: srcpkgs/pcsx2/template [hunk 2]
```diff
@@ -4,19 +4,20 @@ lib32mode="full"
...
 build_style=cmake
 configure_args="-DGLSL_API=1 -DSDL2_API=0 -DPACKAGE_MODE=1 -DREBUILD_SHADER=1 -DXDG_STD=1
  -DPLUGIN_DIR=/usr/lib32/pcsx2 -DGAMEINDEX_DIR=/usr/share/pcsx2 -DDISABLE_ADVANCE_SIMD=1
- -DSDL2_API=TRUE"
+ -DSDL2_API=TRUE -DCMAKE_BUILD_TYPE=Release"
```
Comment (abenson) [inline srcpkgs/pcsx2/template:12]: This is already set by cmake build_style

**Comment by abenson** [comment]
It itself is GPL-3.0-or-later, but it includes 3rd party stuff that has various licenses

**Comment by selfsigned** [comment]
Seems like this qualifies as a 'huge package', the job length is too long for travis.

File: srcpkgs/pcsx2/template (hunk 1 above)
Comment (selfsigned) [reply srcpkgs/pcsx2/template]: Whoops, fixed

**Comment by abenson** [comment]
(PCSX2:25266): Gtk-WARNING **: 13:33:13.451: Error loading theme icon 'document-save' for stock: Unable to load image-loading module: /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-png.so: /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-png.so: wrong ELF class: ELFCLASS64

Looks like it's trying to load things from the wrong path.

File: srcpkgs/pcsx2/template [hunk 3]
```diff
@@ -4,19 +4,20 @@ lib32mode="full"
 nopie=yes
 
 pkgname=pcsx2
-version=1.4.0
-revision=7
+version=1.6.0
+revision=1
 build_style=cmake
 configure_args="-DGLSL_API=1 -DSDL2_API=0 -DPACKAGE_MODE=1 -DREBUILD_SHADER=1 -DXDG_STD=1
```
Comment (abenson) [inline srcpkgs/pcsx2/template:10]: `-DSDL2_API=0` should be replaced with the `-DSDL2_API=TRUE`

**Comment by abenson** [comment]
Ah, doesn't seem to be an issue with PCSX2.

**Pull request merged**